src/commands/binder.ts

```diff
@@ -31,3 +38,22 @@ export function registerCommandSetThemeDay(
     setTheme(context, true);
   });
 }
+
+/**
+ * Creates and returns the Light Switch Status Bar Item
+ *
+ * Note: The returned StatusBarItem still needs to be pushed into vscode's subscriptions
+ */
+export function createLightSwitchStatusBarItem(): StatusBarItem {
+  const statusBarItem = window.createStatusBarItem(
+    StatusBarAlignment.Left,
+    100 // The higher the number the farther left it is on the status bar
```

Comment:
1. Could you set the type for `statusBarItem`? It should be `StatusBarItem`.
2. Could you `100` into a const variable called `priority`?